cmd-script.c

```diff
@@ -19,6 +19,15 @@
 
 #include "libtraceevent/event-parse.h"
 
+#if !HAVE_LIBPYTHON2
```

Comment:
I'd like to make the cmd-script.c file agnostic about python support.  I think current code is ok as long as python init code returns an error if not available.  Please see my comment on the earlier commit.